make lint is failing on v4.4.2/v4.4.3

**Describe the bug**
Running `make lint` after building v4.4.2 is failing with the below errors (it works as expected against v4.4.1 though).

**Expected behavior**
Functional lint test.

**Logs**
```
yarn lint
yarn run v1.22.19
$ eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/specs/dbConnector.test.ts
   9:24  error  Insert `,`  prettier/prettier
  20:34  error  Insert `,`  prettier/prettier

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/specs/timeseries.spec.ts
   5:1   error  Delete `⏎`                                                                                                  prettier/prettier
   9:10  error  Replace `[0,·1],·[1,·2],·[1,·3]` with `⏎··········[0,·1],⏎··········[1,·2],⏎··········[1,·3],⏎········`     prettier/prettier
  10:10  error  Replace `[2,·1],·[3,·2],·[4,·3]]` with `⏎··········[2,·1],⏎··········[3,·2],⏎··········[4,·3],⏎········],`  prettier/prettier
  13:23  error  Replace `[2,·1],·[4,·2],·[5,·3]` with `⏎········[2,·1],⏎········[4,·2],⏎········[5,·3],⏎······`             prettier/prettier
  23:10  error  Replace `[1,·1],·[1,·2],·[1,·3]` with `⏎··········[1,·1],⏎··········[1,·2],⏎··········[1,·3],⏎········`     prettier/prettier
  24:10  error  Replace `[3,·2],·[4,·3]]` with `⏎··········[3,·2],⏎··········[4,·3],⏎········],`                            prettier/prettier
  27:23  error  Replace `[1,·1],·[4,·2],·[5,·3]` with `⏎········[1,·1],⏎········[4,·2],⏎········[5,·3],⏎······`             prettier/prettier
  35:21  error  Replace `[1,·1],·[-4,·2],·[2,·3]` with `⏎········[1,·1],⏎········[-4,·2],⏎········[2,·3],⏎······`           prettier/prettier
  37:23  error  Replace `[101,·1],·[96,·2],·[102,·3]` with `⏎········[101,·1],⏎········[96,·2],⏎········[102,·3],⏎······`   prettier/prettier

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/specs/utils.spec.ts
    5:1   error  Replace `⏎··describe('expandItemName()',·()·=>·{⏎` with `··describe('expandItemName()',·()·=>·{`                                                                                                                   prettier/prettier
   13:21  error  Replace `"CPU·user·time"` with `'CPU·user·time',`                                                                                                                                                                  prettier/prettier
   18:21  error  Replace `"CPU·system·time·-·avg1"` with `'CPU·system·time·-·avg1',`                                                                                                                                                prettier/prettier
   23:21  error  Replace `"CPU·-··-·system·-·avg1"` with `'CPU·-··-·system·-·avg1',`                                                                                                                                                prettier/prettier
   24:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   27:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
   39:21  error  Replace `"CPU·user·time"` with `'CPU·user·time',`                                                                                                                                                                  prettier/prettier
   44:21  error  Replace `"CPU·type=user,value=avg·time"` with `'CPU·type=user,value=avg·time',`                                                                                                                                    prettier/prettier
   49:21  error  Replace `"CPU·type=user,value=avg·time·user"` with `'CPU·type=user,value=avg·time·user',`                                                                                                                          prettier/prettier
   54:21  error  Replace `"CPU·type=user,value=avg·time·user"` with `'CPU·type=user,value=avg·time·user',`                                                                                                                          prettier/prettier
   55:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   58:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
   70:21  error  Replace `"CPU·user,system·-·avg1·time"` with `'CPU·user,system·-·avg1·time',`                                                                                                                                      prettier/prettier
   75:59  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   80:59  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   81:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   84:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
   92:43  error  Delete `⏎`                                                                                                                                                                                                         prettier/prettier
   99:63  error  Replace `\w|\d)*-alu-[0-9{2}/']` with `w|d)*-alu-[0-9{2}/'],`                                                                                                                                                      prettier/prettier
  103:41  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  104:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  107:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
  118:61  error  Replace `\w|\d)*-alu-[0-9]*/']` with `w|d)*-alu-[0-9]*/'],`                                                                                                                                                        prettier/prettier
  122:61  error  Replace `\w|\d)*-alu-[0-9]{2}/']` with `w|d)*-alu-[0-9]{2}/'],`                                                                                                                                                    prettier/prettier
  126:41  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  130:35  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  131:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  134:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
  147:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  151:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  154:19  error  Replace `[1,·2],·[3,·4]` with `⏎············[1,·2],⏎············[3,·4],⏎··········`                                                                                                                                prettier/prettier
  155:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  158:19  error  Replace `[[1,·2],·[3,·4]],·[[1,·2],·[3,·4]]` with `⏎············[⏎··············[1,·2],⏎··············[3,·4],⏎············],⏎············[⏎··············[1,·2],⏎··············[3,·4],⏎············],⏎··········`  prettier/prettier
  159:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/zabbix/connectors/zabbix_api/types.ts
  22:37  error  Replace `[key:·string]:·any` with `·[key:·string]:·any·`  prettier/prettier
  26:33  error  Replace `[key:·string]:·any` with `·[key:·string]:·any·`  prettier/prettier

✖ 47 problems (47 errors, 0 warnings)
  47 errors and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [Makefile:72: lint] Error 1
```

I remain available if you need any more information :)

--- a/src/datasource/specs/dbConnector.test.ts
+++ b/src/datasource/specs/dbConnector.test.ts
@@ -6,7 +6,7 @@ const getAllMock = jest.fn().mockReturnValue([{ id: 42, name: 'foo', meta: {} }]
 jest.mock('@grafana/runtime', () => ({
   getDataSourceSrv: () => ({
     get: loadDatasourceMock,
-    getList: getAllMock
+    getList: getAllMock,
   }),
 }));
 
@@ -17,7 +17,7 @@ describe('DBConnector', () => {
     beforeEach(() => {
       ctx.options = {
         datasourceId: 42,
-        datasourceName: undefined
+        datasourceName: undefined,
       };
 
       loadDatasourceMock.mockClear();

--- a/src/datasource/specs/timeseries.spec.ts
+++ b/src/datasource/specs/timeseries.spec.ts
@@ -2,15 +2,26 @@
 import ts from '../timeseries';
 
 describe('timeseries processing functions', () => {
-
   describe('sumSeries()', () => {
     it('should properly sum series', (done) => {
       let series = [
-        [[0, 1], [1, 2], [1, 3]],
-        [[2, 1], [3, 2], [4, 3]]
+        [
+          [0, 1],
+          [1, 2],
+          [1, 3],
+        ],
+        [
+          [2, 1],
+          [3, 2],
+          [4, 3],
+        ],
       ];
 
-      let expected = [[2, 1], [4, 2], [5, 3]];
+      let expected = [
+        [2, 1],
+        [4, 2],
+        [5, 3],
+      ];
 
       let result = ts.sumSeries(series);
       expect(result).toEqual(expected);
@@ -20,11 +31,22 @@ describe('timeseries processing functions', () => {
     it('should properly sum series with nulls', (done) => {
       // issue #286
       let series = [
-        [[1, 1], [1, 2], [1, 3]],
-        [[3, 2], [4, 3]]
+        [
+          [1, 1],
+          [1, 2],
+          [1, 3],
+        ],
+        [
+          [3, 2],
+          [4, 3],
+        ],
       ];
 
-      let expected = [[1, 1], [4, 2], [5, 3]];
+      let expected = [
+        [1, 1],
+        [4, 2],
+        [5, 3],
+      ];
 
       let result = ts.sumSeries(series);
       expect(result).toEqual(expected);
@@ -32,9 +54,17 @@ describe('timeseries processing functions', () => {
     });
 
     it('should properly offset metric', (done) => {
-      let points = [[1, 1], [-4, 2], [2, 3]];
+      let points = [
+        [1, 1],
+        [-4, 2],
+        [2, 3],
+      ];
 
-      let expected = [[101, 1], [96, 2], [102, 3]];
+      let expected = [
+        [101, 1],
+        [96, 2],
+        [102, 3],
+      ];
 
       let result = ts.offset(points, 100);
       expect(result).toEqual(expected);

--- a/src/datasource/specs/utils.spec.ts
+++ b/src/datasource/specs/utils.spec.ts
@@ -92,8 +92,8 @@ describe('Utils', () => {
     it('should properly split query in old format', (done) => {
       let test_cases = [
         {
-          query: `/alu/./tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/`,
-          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9{2}/'],
+          query: `/alu/./tw-(nyc|que|brx|dwt|brk)-sta_(\\w|\\d)*-alu-[0-9{2}/`,
+          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(\\w|\\d)*-alu-[0-9{2}/'],
         },
         {
           query: `a.b.c.d`,
@@ -111,12 +111,12 @@ describe('Utils', () => {
     it('should properly split query', (done) => {
       let test_cases = [
         {
-          query: `{alu}{/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9]*/}`,
-          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9]*/'],
+          query: `{alu}{/tw-(nyc|que|brx|dwt|brk)-sta_(\\w|\\d)*-alu-[0-9]*/}`,
+          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(\\w|\\d)*-alu-[0-9]*/'],
         },
         {
-          query: `{alu}{/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9]{2}/}`,
-          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9]{2}/'],
+          query: `{alu}{/tw-(nyc|que|brx|dwt|brk)-sta_(\\w|\\d)*-alu-[0-9]{2}/}`,
+          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(\\w|\\d)*-alu-[0-9]{2}/'],
         },
         {
           query: `{a}{b}{c}{d}`,

--- a/src/datasource/specs/utils.spec.ts
+++ b/src/datasource/specs/utils.spec.ts
@@ -2,29 +2,27 @@ import _ from 'lodash';
 import * as utils from '../utils';
 
 describe('Utils', () => {
-
   describe('expandItemName()', () => {
-
     it('should properly expand unquoted params', (done) => {
       let test_cases = [
         {
           name: `CPU $2 time`,
           key: `system.cpu.util[,user,avg1]`,
-          expected: "CPU user time"
+          expected: 'CPU user time',
         },
         {
           name: `CPU $2 time - $3`,
           key: `system.cpu.util[,system,avg1]`,
-          expected: "CPU system time - avg1"
+          expected: 'CPU system time - avg1',
         },
         {
           name: `CPU - $1 - $2 - $3`,
           key: `system.cpu.util[,system,avg1]`,
-          expected: "CPU -  - system - avg1"
-        }
+          expected: 'CPU -  - system - avg1',
+        },
       ];
 
-      _.each(test_cases, test_case => {
+      _.each(test_cases, (test_case) => {
         let expandedName = utils.expandItemName(test_case.name, test_case.key);
         expect(expandedName).toBe(test_case.expected);
       });
@@ -36,26 +34,26 @@ describe('Utils', () => {
         {
           name: `CPU $2 time`,
           key: `system.cpu.util["type=user,value=avg",user]`,
-          expected: "CPU user time"
+          expected: 'CPU user time',
         },
         {
           name: `CPU $1 time`,
           key: `system.cpu.util["type=user,value=avg","user"]`,
-          expected: "CPU type=user,value=avg time"
+          expected: 'CPU type=user,value=avg time',
         },
         {
           name: `CPU $1 time $3`,
           key: `system.cpu.util["type=user,value=avg",,"user"]`,
-          expected: "CPU type=user,value=avg time user"
+          expected: 'CPU type=user,value=avg time user',
         },
         {
           name: `CPU $1 $2 $3`,
           key: `system.cpu.util["type=user,value=avg",time,"user"]`,
-          expected: "CPU type=user,value=avg time user"
-        }
+          expected: 'CPU type=user,value=avg time user',
+        },
       ];
 
-      _.each(test_cases, test_case => {
+      _.each(test_cases, (test_case) => {
         let expandedName = utils.expandItemName(test_case.name, test_case.key);
         expect(expandedName).toBe(test_case.expected);
       });
@@ -67,21 +65,21 @@ describe('Utils', () => {
         {
           name: `CPU $2 - $3 time`,
           key: `system.cpu.util[,[user,system],avg1]`,
-          expected: "CPU user,system - avg1 time"
+          expected: 'CPU user,system - avg1 time',
         },
         {
           name: `CPU $2 - $3 time`,
           key: `system.cpu.util[,["user,system",iowait],avg1]`,
-          expected: `CPU "user,system",iowait - avg1 time`
+          expected: `CPU "user,system",iowait - avg1 time`,
         },
         {
           name: `CPU - $2 - $3 - $4`,
           key: `system.cpu.util[,[],["user,system",iowait],avg1]`,
-          expected: `CPU -  - "user,system",iowait - avg1`
-        }
+          expected: `CPU -  - "user,system",iowait - avg1`,
+        },
       ];
 
-      _.each(test_cases, test_case => {
+      _.each(test_cases, (test_case) => {
         let expandedName = utils.expandItemName(test_case.name, test_case.key);
         expect(expandedName).toBe(test_case.expected);
       });
@@ -90,21 +88,20 @@ describe('Utils', () => {
   });
 
   describe('splitTemplateQuery()', () => {
-
     // Backward compatibility
     it('should properly split query in old format', (done) => {
       let test_cases = [
         {
           query: `/alu/./tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/`,
-          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/']
+          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9{2}/'],
         },
         {
           query: `a.b.c.d`,
-          expected: ['a', 'b', 'c', 'd']
-        }
+          expected: ['a', 'b', 'c', 'd'],
+        },
       ];
 
-      _.each(test_cases, test_case => {
+      _.each(test_cases, (test_case) => {
         let splitQuery = utils.splitTemplateQuery(test_case.query);
         expect(splitQuery).toEqual(test_case.expected);
       });
@@ -115,23 +112,23 @@ describe('Utils', () => {
       let test_cases = [
         {
           query: `{alu}{/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9]*/}`,
-          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9]*/']
+          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9]*/'],
         },
         {
           query: `{alu}{/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9]{2}/}`,
-          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9]{2}/']
+          expected: ['alu', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9]{2}/'],
         },
         {
           query: `{a}{b}{c}{d}`,
-          expected: ['a', 'b', 'c', 'd']
+          expected: ['a', 'b', 'c', 'd'],
         },
         {
           query: `{a}{b.c.d}`,
-          expected: ['a', 'b.c.d']
-        }
+          expected: ['a', 'b.c.d'],
+        },
       ];
 
-      _.each(test_cases, test_case => {
+      _.each(test_cases, (test_case) => {
         let splitQuery = utils.splitTemplateQuery(test_case.query);
         expect(splitQuery).toEqual(test_case.expected);
       });
@@ -144,19 +141,31 @@ describe('Utils', () => {
       const test_cases = [
         {
           array: [],
-          depth: 1
+          depth: 1,
         },
         {
           array: [1, 2, 3],
-          depth: 1
+          depth: 1,
         },
         {
-          array: [[1, 2], [3, 4]],
-          depth: 2
+          array: [
+            [1, 2],
+            [3, 4],
+          ],
+          depth: 2,
         },
         {
-          array: [[[1, 2], [3, 4]], [[1, 2], [3, 4]]],
-          depth: 3
+          array: [
+            [
+              [1, 2],
+              [3, 4],
+            ],
+            [
+              [1, 2],
+              [3, 4],
+            ],
+          ],
+          depth: 3,
         },
       ];
 

--- a/src/datasource/zabbix/connectors/zabbix_api/types.ts
+++ b/src/datasource/zabbix/connectors/zabbix_api/types.ts
@@ -19,11 +19,11 @@ export interface JSONRPCError {
   data?: string;
 }
 
-export type JSONRPCRequestParams = {[key: string]: any};
+export type JSONRPCRequestParams = { [key: string]: any };
 
 export type HTTPMethod = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE' | 'HEAD' | 'CONNECT' | 'OPTIONS' | 'TRACE';
 
-export type GFRequestOptions = {[key: string]: any};
+export type GFRequestOptions = { [key: string]: any };
 
 export interface ZabbixRequestResponse {
   data?: JSONRPCResponse<any>;

